requirements.txt

```diff
@@ -2,6 +2,7 @@ Django==3.1.5
 django-crispy-forms==1.10.0
 django-impersonate==1.7.2
 django-polymorphic==3.0.0
+faker==5.8.0
```

Comment:
this way the dependency will also be installed on produktive systems which makes no sense and just uses up valluable space.